libraries/chain/protocol/transaction.cpp

```diff
@@ -359,6 +359,42 @@ set<public_key_type> signed_transaction::get_required_signatures(
    return result;
 }
 
+void signed_transaction::get_required_signatures(
```

Comment:
Your new method is mostly a copy of existing code. Please keep your code [DRY](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&ved=2ahUKEwjMnZi8l53fAhXDKVAKHR_kAgYQFjAAegQIDBAB&url=https%3A%2F%2Fde.wikipedia.org%2Fwiki%2FDon%25E2%2580%2599t_repeat_yourself&usg=AOvVaw1dDftLLylrw8QicbziCNcl).

Reply:
got it